python/HeaderService/hutils.py

```diff
@@ -310,22 +326,22 @@ def load_templates(self):
         self.header_primary = read_head_template(self.templ_primary_file)
 
         # Load up the template for the PRIMARY header
-        LOGGER.info("Loading template for: {}".format('PRIMARY'))
+        self.log.info("Loading template for: {}".format('PRIMARY'))
```

Comment:
Consider `self.log.info("Loading template for PRIMARY")`

format is overkill for a fixed string.

Reply:
Fixed. Thanks.